contracts/mainnet/connectors/compound/v3-rewards/main.sol

```diff
@@ -0,0 +1,75 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title Compound.
+ * @dev Rewards.
+ */
+
+import { TokenInterface } from "../../../common/interfaces.sol";
+import { Stores } from "../../../common/stores.sol";
+import { Helpers } from "./helpers.sol";
+import { Events } from "./events.sol";
+
+abstract contract CompoundV3RewardsResolver is Events, Helpers {
+	/**
+	 * @dev Claim rewards and interests accrued in supplied/borrowed base asset.
+	 * @notice Claim rewards and interests accrued.
+	 * @param market The address of the market.
+	 * @param account The account of which the rewards are to be claimed.
+	 * @param accrue Should accrue the rewards and interest before claiming.
+	 * @param setId ID stores the amount of rewards claimed.
+	 */
+	function claimRewards(
+		address market,
+		address account,
+		bool accrue,
+		uint256 setId
+	) public returns (string memory eventName_, bytes memory eventParam_) {
+		uint256 rewardsOwed = cometRewards.getRewardOwed(market, account).owed;
+		cometRewards.claim(market, account, accrue);
+
+		setUint(setId, rewardsOwed);
+
+		eventName_ = "LogRewardsClaimed(address,address,uint256,uint256,bool)";
+		eventParam_ = abi.encode(market, account, rewardsOwed, setId, accrue);
+	}
+
+	/**
+	 * @dev Claim rewards and interests accrued in supplied/borrowed base asset.
+	 * @notice Claim rewards and interests accrued and transfer to dest address.
+	 * @param market The address of the market.
+	 * @param account The account of which the rewards are to be claimed.
+	 * @param dest The account where to transfer the claimed rewards.
+	 * @param accrue Should accrue the rewards and interest before claiming.
+	 * @param setId ID stores the amount of rewards claimed.
+	 */
+	function claimRewardsTo(
+		address market,
+		address account,
+		address dest,
+		bool accrue,
+		uint256 setId
+	) public returns (string memory eventName_, bytes memory eventParam_) {
+		//in reward token decimals
+		uint256 rewardsOwed = cometRewards.getRewardOwed(market, account).owed;
```

Comment:
@Richa-iitr This gives the claimable amount? Or claimed + claimable amount?

Reply:
claimable

Reply:
Its calaculated like this: https://github.com/compound-finance/comet/blob/main/contracts/CometRewards.sol#L119

Reply:
Okay